borg/helpers.py

```diff
@@ -98,6 +102,15 @@ def check_extension_modules():
         raise ExtensionModuleError
 
 
+SUPPORTED_REPO_FEATURES = frozenset()
+
+REPO_OPERATION_READ = 'read'
+REPO_OPERATION_CHECK = 'check'
+REPO_OPERATION_WRITE = 'write'
+REPO_OPERATION_DELETE = 'delete'
+REPO_OPERATION_NO_CHECK = '-'
```

Comment:
Now implemented with the stdlib enum class/metaclass. I think requiring 3.4 is ok? Or do we still target 3.3?